src/About/Feedback.tsx

```diff
@@ -136,13 +149,18 @@ const FeedbackDialog = ({
 const handleFormData = async (
     feedback: string,
     setResponse: (response: boolean) => void,
-    category?: string
+    category?: string,
+    onError?: () => void
```

Comment:
- What is the purpose of this callback, is there already a plan to use it somewhere (for what?)? 
- Would it be helpful to also pass the error to it?
- Should its addition be mentioned in the changelog?

Reply:
1) currently there is no way to know that any error happened outside of the handleFormData(). I could throw error next and catch later where the hook is called, but this sounds unsafe if hook is used elsewhere.
2) True, but looks unnecessary for this use case, can be added later if needed?
3) It is optional, doesn't break anything and most likely not needed for others, so I think no? Here I'm not sure